spec.html

```diff
@@ -16277,179 +16277,48 @@ <h2>Syntax</h2>
     <p>A line terminator can occur within a |MultiLineComment| but cannot occur within a |SingleLineComment|.</p>
     <p>Line terminators are included in the set of white space code points that are matched by the `\\s` class in regular expressions.</p>
-    <p>The ECMAScript line terminator code points are listed in <emu-xref href="#table-line-terminator-code-points"></emu-xref>.</p>
-    <emu-table id="table-line-terminator-code-points" caption="Line Terminator Code Points" oldids="table-33">
-      <table>
-        <tr>
-          <th>
-            Code Point
-          </th>
-          <th>
-            Unicode Name
-          </th>
-          <th>
-            Abbreviation
-          </th>
-        </tr>
-        <tr>
-          <td>
-            `U+000A`
-          </td>
-          <td>
-            LINE FEED (LF)
-          </td>
-          <td>
-            &lt;LF>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+000D`
-          </td>
-          <td>
-            CARRIAGE RETURN (CR)
-          </td>
-          <td>
-            &lt;CR>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+2028`
-          </td>
-          <td>
-            LINE SEPARATOR
-          </td>
-          <td>
-            &lt;LS>
-          </td>
-        </tr>
-        <tr>
-          <td>
-            `U+2029`
-          </td>
-          <td>
-            PARAGRAPH SEPARATOR
-          </td>
-          <td>
-            &lt;PS>
-          </td>
-        </tr>
-      </table>
-    </emu-table>
-    <p>Only the Unicode code points in <emu-xref href="#table-line-terminator-code-points"></emu-xref> are treated as line terminators. Other new line or line breaking Unicode code points are not treated as line terminators but are treated as white space if they meet the requirements listed in <emu-xref href="#table-white-space-code-points"></emu-xref>. The sequence &lt;CR>&lt;LF> is commonly used as a line terminator. It should be considered a single |SourceCharacter| for the purpose of reporting line numbers.</p>
+    <p>Only the Unicode code point sequences matched by |LineTerminatorSequence| are treated as line terminators. Other new line or line breaking Unicode code points are not treated as line terminators but are treated as white space if they are matched by |WhiteSpace|. The sequence « U+000D (CARRIAGE RETURN), U+000A (LINE FEED) » is commonly used as a line terminator. It should be considered a single |SourceCharacter| for the purpose of reporting line numbers.</p>
     <h2>Syntax</h2>
     <emu-grammar type="definition">
       LineTerminator ::
-        &lt;LF&gt;
-        &lt;CR&gt;
-        &lt;LS&gt;
-        &lt;PS&gt;
```

Comment:
There are remaining occurrences of
 `&lt;(LF|CR|LS|PS)&gt;`
that have not been converted to the new form.